examples/js/controls/DragControls.js

```diff
@@ -89,7 +95,7 @@ THREE.DragControls = function ( _objects, _camera, _domElement ) {
 
 			var object = intersects[ 0 ].object;
 
-			_plane.setFromNormalAndCoplanarPoint( _camera.getWorldDirection( _plane.normal ), object.position );
+			_plane.setFromNormalAndCoplanarPoint( _planeNormal || _camera.getWorldDirection( _plane.normal ) , object.position );
```

Comment:
@mrdoob If I initialize `_planeNormal` as `THREE.Vector3`, how would you suggest modifying the above line?

Reply:
Oh, I see...